examples/fib-debug/main.c

```diff
@@ -116,3 +119,15 @@ int main(int argc, const char* argv[]) {
   return 0;
 }
 
+static void print_error(const char *message, wasmtime_error_t *error, wasm_trap_t *trap) {
+  fprintf(stderr, "error: %s\n", message);
+  wasm_byte_vec_t error_message;
+  if (error != NULL) {
+    wasmtime_error_message(error, &error_message);
+  } else {
+    wasm_trap_message(trap, &error_message);
+  }
+  fprintf(stderr, "%.*s\n", (int) error_message.size, error_message.data);
+  wasm_byte_vec_delete(&error_message);
+  exit(1);
```

Comment:
Nit: I realize this is example code (although the amount of example code that ends up in use is always surprising), but it's a bit strange, at least to me, for a function called `print_error` to exit.  Perhaps `exit_with_error` or something similar?